recipe/meta.yaml

```diff
@@ -1,24 +1,25 @@
-{% set version = "0.7.4" %}
+{% set name = "creoleparser" %}
+{% set version = "0.7.5" %}
 
 package:
-  name: creoleparser
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/C/Creoleparser/Creoleparser-{{ version }}.zip
-  sha256: 07af317e5c19e819130e01d8d698a741f05d69572642f5710b22b9f3630b45f3
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/Creoleparser-{{ version }}.tar.gz
+  sha256: 62999c115f3ece801ff31cdde7cab8f41df4ed95c189e1c6f02e7a036733a641
 
 build:
   noarch: python
   number: 0
-  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  script: {{ PYTHON }} -m pip install . -vv
 
 requirements:
-  build:
-    - python <3
+  host:
+    - python >=3.6
     - pip 
   run:
-    - python <3
+    - python >=3.6
```

Comment:
Does this work beyond importing on python 3? I cannot tell from the upstream repo, though I do see commits from 2019.

Reply:
In the setup.py they seem to support both https://github.com/wikiotics/creoleparser/blob/master/setup.py#L37

Reply:
Ah good call! I missed that bit. Thank you!